docs/features/powershell/azure-sql.md

```diff
@@ -0,0 +1,52 @@
+---
```

Comment:
This change should be done in the `./preview` folder, not in the published folder.